.travis.yml

```diff
@@ -1,49 +1,66 @@
 language: php
-
 sudo: false
-
 cache:
     directories:
         - $HOME/.composer/cache/files
         - $HOME/symfony-bridge/.phpunit
 
+env:
+    global:
+        - PHPUNIT_FLAGS="-v"
+        - SYMFONY_PHPUNIT_DIR="$HOME/symfony-bridge/.phpunit"
+        - DEPENDENCIES="zendframework/zend-diactoros:^1.4.1"
+
 matrix:
+    fast_finish: true
     include:
+          # Minimum supported dependencies with the latest and oldest PHP version
+        - php: 7.2
+          env: COMPOSER_FLAGS="--prefer-stable --prefer-lowest" SYMFONY_DEPRECATIONS_HELPER="weak_vendors"
         - php: 5.3
           dist: 'precise'
-        - php: 5.4
-        - php: 5.5
-        - php: 5.6
+          env: COMPOSER_FLAGS="--prefer-stable --prefer-lowest" SYMFONY_DEPRECATIONS_HELPER="weak_vendors" DEPENDENCIES=""
+
+          # Test the latest stable release
         - php: 5.3
           dist: 'precise'
-          env: COMPOSER_OPTIONS="--prefer-lowest --prefer-stable" SYMFONY_DEPRECATIONS_HELPER=weak
+          env: DEPENDENCIES=""
+        - php: 5.4
+        - php: 5.5
         - php: 5.6
-          env: COMPOSER_OPTIONS="" SYMFONY_DEPRECATIONS_HELPER=weak
         - php: 7.0
-        - php: hhvm
-    allow_failures:
-        - php: hhvm
-    fast_finish: true
+        - php: 7.1
+        - php: 7.2
+          env: COVERAGE=true PHPUNIT_FLAGS="-v --coverage-text"
 
-env:
-    global:
-        - deps=no
-        - SYMFONY_DEPRECATIONS_HELPER=strict
-        - SYMFONY_PHPUNIT_DIR="$HOME/symfony-bridge/.phpunit"
-        - COMPOSER_OPTIONS="--prefer-stable"
+          # Test LTS versions. This makes sure we do not use Symfony packages with version greater
+          # than 2 or 3 respectively.
+        - php: 7.2
+          env: DEPENDENCIES="symfony/lts:^2 symfony/force-lowest:^2.8 zendframework/zend-diactoros:^1.4.1"
```

Comment:
symfony/force-lowest should reference an exact version to work correctly:
symfony/force-lowest:=2.8 or symfony/force-lowest:v2.8